ckanext/gla/templates/about/frequently-asked-questions.html

```diff
@@ -0,0 +1,26 @@
+{% extends "page.html" %}
+
+{% block main_content %}
+<div class="module-content">
+    <h1>Frequently Asked Questions</h1>
+    <h2>What is the London Datastore?</h2>
+    <p>The London Datastore is an initiative by the Greater London Authority (GLA) to release as much of the data that it holds as possible.</p>
+    <h2>What data cannot or will not be released?</h2>
+    <p>Our policy is to only withhold data where to release it would infringe privacy legislation or a contractual obligation eg commercial confidentiality.</p>
+    <h2>Isn’t some of this data available elsewhere?</h2>
+    <p>We recognise that some organisations already provide open access to their data and where appropriate we have provided a direct link to their website, all available from one place.&nbsp; However we feel that there are some instances where we can add value to this, for example by focussing on just the data for London, by saving you from having to download separate datasets for multiple years, or by making it available in a standardised and easy to use format.</p>
+    <h2>Will the Datastore duplicate information held in London.gov.uk website?</h2>
+    <p>Whilst this is possible, the idea behind the Datastore is to provide raw data that the GLA holds, leaving the main website of the GLA, www.london.gov.uk to make available information in its final state (eg. Policy and Strategy documents).</p>
```

Comment:
Should this be an href? Github didn't highlight the line properly, but the `www.london.gov.uk` part above